galactic/distribution.yaml

```diff
@@ -1291,6 +1291,13 @@ repositories:
       url: https://github.com/MRPT/mrpt.git
       version: develop
     status: developed
+  nao_interfaces:
+    release:
```

Comment:
Consider adding a `source` entry.

Reply:
I'm not too certain on how I could do this change with bloom without opening a new PR.
Would manually adding this to ``ijnek:bloom-nao_interfaces-0`` branch be appropriate?

```
    source:
      type: git
      url: https://github.com/ijnek/nao_interfaces.git
```

Reply:
You can manually push changes to this PR.

Reply:
Great, done.